### PR TITLE
Gatewatcher lis generic command

### PR DESCRIPTION
**Pull request by Liolaeus**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Gatewatcher-LIS/Integrations/LastInfoSec/LastInfoSec.py [hunk 1]
```diff
@@ -334,6 +589,13 @@ def main() -> None:
     token = params.get("token")
     check_cert = params.get("check_cert", False)
     proxy = params.get("proxy", False)
+    reliability = params.get('integrationReliability')
```
Comment (ilappe) [inline Packs/Gatewatcher-LIS/Integrations/LastInfoSec/LastInfoSec.py:592]: you can do a small refactoring here after you use the type 15
(see my comment in the definition of `integrationReliability` in the YML fiile)

File: Packs/Gatewatcher-LIS/Integrations/LastInfoSec/LastInfoSec.yml [hunk 2]
```diff
@@ -80,42 +85,244 @@ script:
...
-  dockerimage: demisto/python3:3.10.12.63474
+  dockerimage: demisto/python3:3.10.13.75921
   runonce: false
   script: '-'
   subtype: python3
   type: python
+  feed: false
+  isfetch: false
+  longRunning: false
+  longRunningPort: false
```
Comment (ilappe) [inline Packs/Gatewatcher-LIS/Integrations/LastInfoSec/LastInfoSec.yml:325]: I assume this section was added automatically how did you update the yml file?

**Review by ilappe — CHANGES_REQUESTED** [review]
Hi @Liolaeus
Thank you for your contribution!
Very good work, clean implementation :)

please see my small comments 
and feel free to reach out to me with any questions - I'm available here or on slack

File: Packs/Gatewatcher-LIS/Integrations/LastInfoSec/LastInfoSec.yml [hunk 3]
```diff
@@ -16,6 +16,11 @@ configuration:
   name: proxy
   type: 8
   required: false
+- name: integrationReliability
```
[the remark this answers is not in the document]
Comment (Liolaeus) [reply Packs/Gatewatcher-LIS/Integrations/LastInfoSec/LastInfoSec.yml:19]: done

File: Packs/Gatewatcher-LIS/Integrations/LastInfoSec/LastInfoSec.py (hunk 1 above)
Comment (Liolaeus) [reply Packs/Gatewatcher-LIS/Integrations/LastInfoSec/LastInfoSec.py]: done

File: Packs/Gatewatcher-LIS/Integrations/LastInfoSec/LastInfoSec.yml (hunk 2 above)
Comment (Liolaeus) [reply Packs/Gatewatcher-LIS/Integrations/LastInfoSec/LastInfoSec.yml:333]: these lines were generated by demisto-sdk, i believe by the command `demisto-sdk format -ud -nv -i Packs/Gatewatcher-LIS/Integrations/LastInfoSec/LastInfoSec.yml`, however i cannot reproduce their generation by removing then and re running the command (demisto-sdk 1.20.7)

**Comment by ilappe** [comment]
Hi @Liolaeus - thanks!
can you send me a short video shows the changes you made?
(use the new commands in the war room and see the output and context)
You can send it via the DFIR in Slack (Israel Lappe)
thanks again..

**Comment by Liolaeus** [comment]
> Hi @Liolaeus - thanks! can you send me a short video shows the changes you made? (use the new commands in the war room and see the output and context) You can send it via the DFIR in Slack (Israel Lappe) thanks again..

Hello, we just finished making the video, and tried to join the DFIR Slack using the following page: https://start.paloaltonetworks.com/join-our-slack-community , unfortunately, upon receiving the invitation we get the following slack error message: "This link is no longer active To join this workspace, you’ll need to ask the person who originally invited you for a new link."

I'd also like to notice you that we had to update the pack's readme.

**Comment by Liolaeus** [comment]
> Hi @Liolaeus - thanks! can you send me a short video shows the changes you made? (use the new commands in the war room and see the output and context) You can send it via the DFIR in Slack (Israel Lappe) thanks again..

Hello, we just finished making the video, and tried to join the DFIR Slack using the following page: https://start.paloaltonetworks.com/join-our-slack-community , unfortunately, upon receiving the invitation we get the following slack error message: "This link is no longer active To join this workspace, you’ll need to ask the person who originally invited you for a new link."

I'd also like to notice you that we had to update the pack's readme.

**Comment by ilappe** [comment]
> 
> Hello, we just finished making the video, and tried to join the DFIR Slack using the following page: https://start.paloaltonetworks.com/join-our-slack-community , unfortunately, upon receiving the invitation we get the following slack error message: "This link is no longer active To join this workspace, you’ll need to ask the person who originally invited you for a new link."
> 
> I'd also like to notice you that we had to update the pack's readme.

Hi @Liolaeus 
thanks for preparing the video 
you can attach the email here but please be sure it's not contained sensitive information etc.

**Comment by Liolaeus** [comment]
> > Hello, we just finished making the video, and tried to join the DFIR Slack using the following page: https://start.paloaltonetworks.com/join-our-slack-community , unfortunately, upon receiving the invitation we get the following slack error message: "This link is no longer active To join this workspace, you’ll need to ask the person who originally invited you for a new link."
> > I'd also like to notice you that we had to update the pack's readme.
> 
> Hi @Liolaeus thanks for preparing the video you can use a video sharing service such as Google Drive or YouTube and attach the link here but please be sure it's not contained sensitive information etc.

Hello @ilappe, you should be able to get the video at the following link: https://drive.google.com/file/d/1g-vMbK8AtgQcLIk8JDI_Ft3l5I5tA222/view?usp=share_link

**Comment by ilappe** [comment]
Thanks @Liolaeus for the demo!
i'm going to merge it  and i will fix the small validation issue
thanks again..

**Review by ilappe — APPROVED** [review]

**Pull request merged**